app/components/ErrorMessage/ErrorMessage.jsx

```diff
@@ -61,7 +61,11 @@ export default class ErrorMessage extends React.Component {
       case Constants.API_CONNECTION_ERROR:
         return {
           body: (
-            <Button accent="system" href={window.location.pathname}>
+            <Button
+              accent="system"
+              href={window.location.pathname}
+              onClick={window.document.location.reload}
```

Comment:
Not sure I understand this. Why are we linking to the current pathname _and_ reloading the page?

Reply:
Added an explanatory comment